src/utils/json-data.js

```diff
@@ -0,0 +1,135 @@
+const fs = require('fs-extra');
+const path = require('path');
+const reserved = require('reserved-words');
+const shared = require('./shared');
+const glob = require('glob');
+const BUILD_TIME_BINDING = '@buildtime:';
+// Matches {{ stache.jsonData. }} with any number of spaces between the '{{' and 'stache.',
+// any keys following the 'jsonData.', and the closing '}}'
+const angularBindingRegExp = new RegExp(/\{\{\s*stache.jsonData.*?\}\}/g);
+const buildTimeBindingRegExp = new RegExp(`\\{\\{\\s*${BUILD_TIME_BINDING}stache.jsonData.*?\\}\\}`, 'g');
+
+let _globalData;
+
+const getGlobalData = () => {
```

Comment:
I think Global is fine, but I see Steve's point too. We could do something like `getAllJsonData`. That's clearer than `dataObject`.